pallets/nft/src/lib.rs

```diff
@@ -1157,6 +1275,15 @@ impl<T: Config> NFTTrait<T::AccountId, BalanceOf<T>> for Pallet<T> {
 		Ok(class_collection_id == collection_id)
 	}
 
+	fn is_stackable(asset_id: (Self::ClassId, Self::TokenId)) -> Result<bool, DispatchError> {
```

Comment:
This could exhaust the block times if the StackableCollections item is significantly large. We need to have storage of the Stackable Collection.

Reply:
Yes this is quite inefficient implementation . Looking into optimising it now.